core/fed_avg_optimizer.py

```diff
@@ -52,16 +52,23 @@ def __init__(self, initialization_payload):
 		serialized_job = initialization_payload.get('serialized_job')
 		self.job = deserialize_job(serialized_job)
 		optimizer_params = initialization_payload.get('optimizer_params')
-		self.listen_iterations = optimizer_params.get('listen_iterations')
-		self.listen_bound = optimizer_params.get('listen_bound')
+		self.listen_iterations = optimizer_params.get('listen_iterations', 0)
+		self.listen_bound = optimizer_params.get('listen_bound', 2)
```

Comment:
I would consider moving the default value of 2 of `listen_bound` to the `configuration.ini`, since it seems arbitrary.